src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -1326,6 +1563,7 @@ def _find_executable(cls, names):
         for name in names:
             ret = cls._which(name)
             if ret is not None:
+                self._log.info("Found %s"%ret)
```

Comment:
As a general rule: for logging commands, please use
```
logger.info('msg: %s', msg)
```
instead of
```
logger.info('msg: %s' % msg)
```
This avoids Python's string expansion to kick in if logging is disabled (and the strings would thus not be used anyway).  We don't do that everywhere either, but in general try to adhere to that rule...  Thanks!